test/_test_helper.js

```diff
@@ -19,10 +19,34 @@ const createPackageJson = function() {
   fs.writeFileSync('package.json', minimalJson);
 };
 
+const createBowerJson = function() {
+  const minimalJson = `{
+    "name": "brunch-app",
+    "description": "Description",
+    "main": "",
+    "authors": ["Your Name"],
+    "license": "MIT",
+    "homepage": "",
+    "ignore": [
+      "**/.*",
+      "node_modules",
+      "bower_components",
+      "test",
+      "tests"
+    ],
+    "dependencies": {},
+    "devDependencies": {}
+  }`;
+
+  fs.writeFileSync('bower.json', minimalJson);
```

Comment:
Why do we need bower there? Bower is going away and brunch recommends to use npm exclusively for client-side deps.